hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java

```diff
@@ -99,7 +99,7 @@ public Topic(
             @JsonProperty("ack") Ack ack,
             @JsonProperty("trackingEnabled") boolean trackingEnabled,
             @JsonProperty("migratedFromJsonType") boolean migratedFromJsonType,
-            @JsonProperty("schemaVersionAwareSerializationEnabled") boolean schemaVersionAwareSerializationEnabled,
+            @JsonProperty("schemaIdAwareSerializationEnabled") boolean schemaIdAwareSerializationEnabled,
```

Comment:
It's depends from ObjectMapper configuration.. If you set: 
```
objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```
then it will be work. It's question for you: do you want fail on unknown properties? If answer is yes then I will leave field `schemaVersionAwareSerializationEnabled` in `Topic` for backward compatiblity.

Reply:
Right now I will add some tests for it.. Let's see it.

Reply:
In that case I think it should not fail on unknown fields and you can change the field name. I see that ObjectMapper in [hermes-management](https://github.com/allegro/hermes/blob/66a9e126beaef1095b589addbed385443f0416e7/hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/ManagementConfiguration.java#L33) and [hermes-{frontend, consumers}](https://github.com/allegro/hermes/blob/7907afb02503787760505cbfecca48369aa292f8/hermes-common/src/main/java/pl/allegro/tech/hermes/common/di/factories/ObjectMapperFactory.java#L15) already have disabled feature `FAIL_ON_UNKNOWN_PROPERTIES`, so it’s ok